### PR TITLE
Tweaked title and description of JSON APIs and AJAX challenge

### LINKED ISSUE #13586: [beta] Consider changing title for "JSON APIs and Ajax: Trigger Click Events with JavaScript"
Challenge [trigger-click-events-with-javascript](http://beta.freecodecamp.com/en/challenges/json-apis-and-ajax/trigger-click-events-with-javascript) has an issue.

As this challenge is about adding an `onclick` handler for a DOM element, I think the title of this challenge should be changed to "Handle Click Events with JavaScript with the onclick Property".

If this challenge were to cover the topic of triggering click events, it should cover `someDOMElement.click()` instead.

The challenge description could also further explain that the `onclick` property only is one way to handle click events in JS.

```html
<script>
  document.addEventListener('DOMContentLoaded',function(){
    // Add your code below this line
    document.getElementById('getMessage').onclick = event => {};
    
    // Add your code above this line
  });
</script>
<style>
  body {
    text-align: center;
    font-family: "Helvetica", sans-serif;
  }
  h1 {
    font-size: 2em;
    font-weight: bold;
  }
  .box {
    border-radius: 5px;
    background-color: #eee;
    padding: 20px 5px;
  }
  button {
    color: white;
    background-color: #4791d0;
    border-radius: 5px;
    border: 1px solid #4791d0;
    padding: 5px 10px 8px 10px;
  }
  button:hover {
    background-color: #0F5897;
    border: 1px solid #0F5897;
  }
</style>
<h1>Cat Photo Finder</h1> 
<p class="message box">
  The message will go here
</p>
<p>
  <button id="getMessage">
    Get Message
  </button>
</p>

```

### PR DESCRIPTION
**Pull request by adrianwithah**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of freeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [X] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [X] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes #13586 

#### Description

Tweaked title of [JSON APIs and Ajax: Trigger click Events with JavaScript challenge](http://beta.freecodecamp.com/en/challenges/json-apis-and-ajax/trigger-click-events-with-javascript), to suit the topic of the challenge, which is the <code>onclick</code> property. Also tweaked the description to move certain portions to a more appropriate challenge. Finally, added a hint that the <code>onclick</code> event handler is but one of many event handlers. @Greenheart

### CODE REVIEW

File: seed/challenges/04-data-visualization/json-apis-and-ajax.json [hunk 1]
```diff
@@ -170,7 +168,7 @@
       "title": "Get JSON with the JavaScript XMLHttpRequest Method",
       "description": [
         "You can also request data from an external source. This is where APIs come into play.",
-        "Remember that APIs - or Application Programming Interfaces - are tools that computers use to communicate with one another.",
+        "Remember that APIs - or Application Programming Interfaces - are tools that computers use to communicate with one another. You'll learn how to update HTML with the data we get from APIs using a technology called AJAX",
```
Comment (QuincyLarson) [inline seed/challenges/04-data-visualization/json-apis-and-ajax.json]: You left off a period at the end of this sentence.

File: seed/challenges/04-data-visualization/json-apis-and-ajax.json [hunk 2]
```diff
@@ -30,14 +30,12 @@
...
       "description": [
-        "This section covers how to get data from APIs. APIs - or Application Programming Interfaces - are tools that computers use to communicate with one another.",
-        "You'll learn how to update HTML with the data we get from these APIs using a technology called AJAX.",
         "You want your code to execute only once your page has finished loading. For that purpose, you can attach a JavaScript event to the document called <code>DOMContentLoaded</code>. Here's the code that does this:",
         "<blockquote>document.addEventListener('DOMContentLoaded',function() {<br><br>});</blockquote>",
-        "Next, you can implement a click event handler that goes inside of the <code>DOMContentLoaded</code> function by adding the following code:",
-        "<code>document.getElementById('getMessage').onclick=function(){};</code>",
+        "You can implement event handlers that goes inside of the <code>DOMContentLoaded</code> function. You can implement an <code>onclick</code> event handler which triggers when the user clicks on the element with id <code>getMessage</code>, by adding the following code:",
```
Comment (QuincyLarson) [inline seed/challenges/04-data-visualization/json-apis-and-ajax.json]: "goes" should be "go". "which" should be "that".

**Review by QuincyLarson — COMMENTED** [review]
This is almost there.

**Review by raisedadead — CHANGES_REQUESTED** [review]
@hay-dee thanks for the pull request, can you please address the review comments?

File: seed/challenges/04-data-visualization/json-apis-and-ajax.json (hunk 2 above)
Comment (hay-dee) [inline seed/challenges/04-data-visualization/json-apis-and-ajax.json]: Hi @QuincyLarson , thanks for reviewing this PR! Should it be "You can implement event handlers **that** go .." or "You can implement event handlers **which** go .."?

**Comment by hay-dee** [comment]
Hi @raisedadead @QuincyLarson , apologies for the delay. I could not find time to work on this during the weekdays. I have updated the PR, hopefully in a correct manner, to address the changes mentioned. Kindly excuse the multiple amendments as its my first time updating a PR 😅 .

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
@QuincyLarson this LGTM, unless you have any more points to add?

File: seed/challenges/04-data-visualization/json-apis-and-ajax.json [hunk 3]
```diff
@@ -30,14 +30,12 @@
     },
     {
       "id": "587d7fad367417b2b2512be1",
-      "title": "Trigger click Events with JavaScript",
+      "title": "Handle Click Events with Javascript using the onclick property",
```
Comment (raisedadead) [inline seed/challenges/04-data-visualization/json-apis-and-ajax.json]:
```diff 
- Javascript
+ JavaScript
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Minor change.

**Comment by hay-dee** [comment]
@raisedadead PR has been updated to include the change 😄

**Comment by QuincyLarson** [comment]
@hay-dee Awesome. Since @raisedadead has an open review, I'll leave merging this to him :)

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @hay-dee !

**Comment by QuincyLarson** [comment]
@hay-dee thanks for your contribution!
